addons/mail/static/src/core/common/thread_model.js

```diff
@@ -282,6 +283,7 @@ export class Thread extends Record {
     });
     /** @type {string} */
     defaultDisplayMode;
+    shouldScrollToUnread = true;
```

Comment:
`should` looks unnecessary, could simply be `scrollToUnread` (heck, even `scrollUnread`)